nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -199,6 +199,12 @@
           <link linkend="opt-services.moosefs">moosefs</link>.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://github.com/PolyMC/PolyMC">polymc</link>,
+          a free, open source launcher for Minecraft.
```

Comment:
```suggestion
          an open source launcher for Minecraft.
```

Reply:
Not only is this a childish thing to do, you've done it to the wrong bit of software. Nice job.

Reply:
My bad, I just added a suggestion to the multimc description